services/bitrise/bitrise.service.js

```diff
@@ -18,14 +18,44 @@ export default class Bitrise extends BaseJsonService {
     queryParamSchema,
   }
 
-  static examples = [
-    {
-      title: 'Bitrise',
-      namedParams: { appId: '3ff11fe8457bd304', branch: 'master' },
-      queryParams: { token: 'lESRN9rEFFfDq92JtXs_jw' },
-      staticPreview: this.render({ status: 'success' }),
+  static openApi = {
+    '/bitrise/{appId}': {
+      get: {
+        summary: 'Bitrise',
+        parameters: [
+          pathParam({
+            name: 'appId',
+            example: '4a2b10a819d12b67',
+          }),
+          queryParam({
+            name: 'token',
+            example: '859FMDR8QHwabCzwvZK6vQ',
+            required: true,
+          }),
```

Comment:
These are the appId and token for https://github.com/OpenAPITools/openapi-generator. It feels slightly off to use someone else's token, but I guess this is what we were already doing, they're public, and they're already used to generate the same badge in their repository's README. Does anyone feel strongly about this?

Reply:
Yeah its an interesting one. Sometimes we do make the example a valid token, usually if the person who contributed the badge puts one in there.

Sometimes we don't though. For example, you'll notice that the token in the Circle CI example that is also in this PR is `abc123def456`, which is clearly not a vlaid token. So :shrug: If we just fill in a random string or leave it as a non-working token it won't be the only case where we have a non-working example.

I'd probably be more inclined to do that than use one without asking, even if it is in a public repo.

Reply:
What is your take on the live tests though?

Reply:
I've updated the examples with a fake token and have reached out [here ](https://github.com/OpenAPITools/openapi-generator/issues/17608) to see if we can use their public token in our repo.

Reply:
Okay, given the absence of response, trying my luck with another repository: https://github.com/JohnEstropia/CoreStore/issues/496

Reply:
OK. Thanks.

If this PR ends up being the last blocker to finishing https://github.com/badges/shields/issues/9285 I think I'd be in favour of just merging this a non-working token in the example and a failing service test - we can follow it up later.